src/plugins/commander/commands/ucp.js

```diff
@@ -1,21 +1,62 @@
 const Command = require('../structs/Command.js');
+const FormatterPlugin = require('../../fandomizer');
+const XRay = require('x-ray');
```

Comment:
1. It doesn't look like `x-ray` is a dependency of Twinkle's right now,
2. could we perhaps just use `?action=raw` on DEV:UCP and regex for this, and
3. if not, could we at least scrape `?action=render`?

Reply:
Yeah, point 2 would probably be the better option rather than scraping as it's a bit more reliable and allows fetching the comments too.